### PR TITLE
Server_Features: added getName() and hardcoded data in the DataServlets.java file

### PR DESCRIPTION
**Pull request by samuelsandoval1**
Please review when you can!

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -23,10 +26,19 @@
...
+  public void init() {
+    names = new ArrayList<String>(Arrays.asList("Sam", "Hadley", "Jerry"));
+  }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String name = names.get((int) (Math.random() * names.size()));
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Sam Sandoval!</h1>");
+    response.getWriter().println("Hello " + name + "!");
```
Comment (jerrychangusa) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:41]: did you want to keep the h1 tags?

Comment (samuelsandoval1) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:41]: No, I believe that is being removed, its red on their

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -15,6 +15,9 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.ArrayList;
```
[the remark this answers is not in the document]
Comment (samuelsandoval1) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:18]: Ok

[the remark this answers is not in the document]
Comment (samuelsandoval1) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:18]: sort all of it or just that section?

**Pull request merged**